File: 2535_Difference_Between_Element_Sum_and_Digit_Sum_of_an_Array.c
int differenceOfSum(int* nums, int numsSize)
{
    int digitSum = 0;
    int elementSum = 0;

    for(int i=0; i<numsSize; i++)
    {
        elementSum+=nums[i];
        
        if(nums[i]/10 == 0)
        {
            digitSum += nums[i];
        }
        else if(nums[i]/10 != 0)
        {   
            int val = nums[i];
            while(val > 0)
            {
                
                digitSum += val%10;
                val/=10;

            }
        }
    }

    int absolute_sum = digitSum - elementSum; 
    return absolute_sum > 0? absolute_sum: -absolute_sum;
}

File: 2535_Difference_Between_Element_Sum_and_Digit_Sum_of_an_Array.py
class Solution:
    def differenceOfSum(self, nums: List[int]) -> int:
        digitSum = 0
        elementSum = 0

        for i in range(len(nums)):
            elementSum = elementSum + nums[i]
            
            if int(nums[i]/10) == 0:
                digitSum += nums[i];
            
            else:
                val = nums[i]
                while val > 0:
                    digitSum = digitSum + val%10
                    val = int(val/10)


        absolute_sum = digitSum - elementSum
        if absolute_sum > 0:
            return absolute_sum
        else: 
            return -absolute_sum